docs/core/diagnostics/eventpipe.md

```diff
@@ -0,0 +1,82 @@
+
+## Decide whether you should use EventPipe or ETW/LTTng
+
+EventPipe is part of the .NET runtime (CoreCLR) and is designed to work the same way across all the platforms .NET Core supports. This allows tracing tools based on EventPipe such as `dotnet-counters`, `dotnet-gcdump` and `dotnet-trace` to work cross platform seemlessly.
+
+However, because EventPipe is a runtime built-in component, its scope is limited to managed code and the runtime itself only, and cannot be used for tracking some lower level events such as resolving native code stack or getting various kernel events. If you use C/C++ interop in your app or you want to trace the runtime itself (which is written in C++), or want deeper diagnostics into the behavior of the app that requires kernel events (i.e. native thread context switching events) you should use ETW or [perf/LTTng](./trace-perfcollect-lttng.md).
+
+The table below is a summary of the differences between EventPipe and ETW/LTTng.
+
+## Use EventPipe to trace your .NET application
+
+You can use EventPipe to trace your .NET application in many ways:
+
+* Use one of the [diagnostics tools](#tools-using-eventpipe) that are built on top of EventPipe.
+
+* Use [Microsoft.Diagnostics.NETCore.Client](https://github.com/dotnet/diagnostics/blob/master/documentation/diagnostics-client-library-instructions.md) library to write your own tool to configure and start EventPipe sessions yourself.
+
+* Use [environment variables](#trace-using-environment-variables) to start EventPipe.
+
+After you've produced a `nettrace` file that contains your EventPipe events, this can be viewed in [`PerfView`](https://github.com/Microsoft/perfview#perfview-overview) or Visual Studio.
+
+You can also analyze EventPipe traces programmatically with [TraceEvent](https://github.com/Microsoft/perfview/blob/master/documentation/TraceEvent/TraceEventLibrary.md).
+
+### Tools using EventPipe
+
+This is the easiest way to use EventPipe to trace your application. To learn more about how to use each of these tools, refer to each tools' documentation.
+
+* [dotnet-counters](./dotnet-counters.md) lets you monitor/collect various metrics emitted by the .NET runtime and core libraries, as well as custom metrics you can write.
+
+* [dotnet-gcdump](./dotnet-gcdump.md) lets you collect GC heap dump of applications for analyzing your application's managed heap.
+
+* [dotnet-trace](./dotnet-trace.md) lets you collect traces of applications to analyze for performance.
+
+## Trace using environment variables
+
+The preferred mechanism for using EventPipe is to use `dotnet-trace` or `Microsoft.Diagnostics.NETCore.Client` library.
+
+However, you can use the following environment variables to set up an EventPipe session on an app and have it write the trace directly to a file. To stop tracing, you need to exit the application.
```

Comment:
```suggestion
The preferred mechanism for using EventPipe is to use `dotnet-trace` or `Microsoft.Diagnostics.NETCore.Client` library, however, you can use the following environment variables to set up an EventPipe session that writes the trace directly to a file. To stop tracing, you need to exit the application.
```

Reply:
pretty sure `, however,` is a comma-splice...? Let me go look up the grammar again

Reply:
For ease of localization, multiple shorter sentences are usually better than one longer sentence.

Reply:
You're correct.  I was thinking of when you use it as an aside, e.g., "You, however, should do otherwise", but in this case it's a conjunctive adverb.  You need to complete the first clause, e.g., "I said to use a comma before the conjunctive adverb; however, that was incorrect!" 😆.  That's what I get for trying to recall English grammar rules.